test/650_proxy_env_test.sh

```diff
@@ -28,4 +28,10 @@ weave_on $HOST1 stop
 weave_on $HOST1 launch-proxy -H tcp://0.0.0.0:12375 -H unix:///var/run/weave/weave.sock
 check
 
+# Check we can use weave env/config with unix -Hs specified
+weave_on $HOST1 stop
+weave_on $HOST1 launch-proxy -H unix:///var/run/weave/weave.sock
+assert_raises 'run_on '$HOST1' "eval $(weave env) ; docker '$CMD'"'
+assert_raises 'run_on '$HOST1' "docker $(weave config) '$CMD'"'
```

Comment:
These tests are failing with a peculiar error (`program terminated with code 1 instead of -e`); the `-e` is coming from inside `$CMD` - the quotation is incorrect.